src/extension.ts

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 import * as path from 'path';
-import { generateStub } from './stubImplementation';
+import { generateStub, extractConcept } from './stubImplementation';
```

Comment:
`extractConcept` can be part of a separate class. Let's not overload `stubImplementation`